src/geometamaker/models.py

```diff
@@ -378,27 +467,29 @@ def load(cls, filepath):
                        f'geometamaker.')
             raise ValueError(message)
 
-        deprecated_attrs = ['metadata_version', 'mediatype', 'name']
-        for attr in deprecated_attrs:
-            if attr in yaml_dict:
-                warnings.warn(
-                    f'"{attr}" exists in {filepath} but is no longer part of '
-                    f'the geometamaker specification. "{attr}" will be '
-                    f'removed from this document. In the future, presence '
-                    f' of "{attr}" will raise a ValidationError',
-                    category=FutureWarning)
-                del yaml_dict[attr]
-
-        # migrate from 'schema' to 'data_model', if needed.
-        if 'schema' in yaml_dict:
-            warnings.warn(
-                "'schema' has been replaced with 'data_model' as an attribute "
-                "name. In the future, the presence of a 'schema' attribute "
-                "will raise a ValidationError",
-                category=FutureWarning)
-            yaml_dict['data_model'] = yaml_dict['schema']
-            del yaml_dict['schema']
-        return cls(**yaml_dict)
```

Comment:
I removed these migrations because these represent old data models that we actually changed before our first release. So it seems okay to not handle these special cases anymore.